README.md

```diff
@@ -62,6 +62,7 @@ HER2 Contest [60] | Breast | Multiple (H&E, IHC) | [data](https://warwick.ac.uk/
 HEROHE - ECDP2020 [27a], [27b] | Breast | H&E | [data](https://ecdp2020.grand-challenge.org/Home/), [paper](https://arxiv.org/ftp/arxiv/papers/2111/2111.04738.pdf) | Train: 359 (positive: 144, negatives: 215), Test: 150 (positive: 60, negative: 90) | images + binary label | classi | wsi | 20x - 3D Histech Pannoramic 1000 | 2020
 HER2 tumor ROIs [70] | Breast | H&E | [data](https://wiki.cancerimagingarchive.net/pages/viewpage.action?pageId=119702524), [paper](https://www.nature.com/articles/s41379-021-00911-w) | 273 | images + ROIs + label | classi (binary) | patch (512x512) | 20x - Aperio ScanScope | 2022 
 HunCRC [71] | Colon | H&E | [data](https://wiki.cancerimagingarchive.net/pages/viewpage.action?pageId=91357370), [github](https://github.com/qbeer/qupath-binarymask-extension), [github](https://github.com/patbaa/crc_data_paper), [paper](https://www.nature.com/articles/s41597-022-01450-y) | 101,389 patches - 200 wsi (from 200 patients) | images + label | classi (10) | wsi - patch (512x512) | 40x - 3DHistech Pannoramic 1000 | 2022
+IMP-CRS 2024 [81a],[81b],[81c] | Colorectal | H&E | [data](https://rdm.inesctec.pt/dataset/nis-2023-008), [paper](https://www.nature.com/articles/s41698-024-00539-4) | nearly 7,000,000 patches - Train 4433 wsi, Test: 900 wsi | images + label | classi (3) | wsi | 40x - Leica GT450 | 2024
```

Comment:
Where does the 7 millions patches comes from? As far as I saw it is a dataset with WSI only right? If yes I would just remove that. But the rest is great :) thanks for your contribution!

Reply:
Oh yes, it is an estimation from our individual extraction of the patches. But I will remove :)